moveit_setup_assistant/package.xml

```diff
@@ -21,11 +21,13 @@
   <build_depend>moveit_ros_planning</build_depend>
   <build_depend>moveit_ros_visualization</build_depend>
   <build_depend>yaml-cpp</build_depend>
+  <build_depend version_gt="0.3.1">srdfdom</build_depend>
  
   <run_depend>moveit_core</run_depend>
   <run_depend>moveit_ros_planning</run_depend>
   <run_depend>moveit_ros_visualization</run_depend>
   <run_depend>yaml-cpp</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend version_gt="0.3.1">srdfdom</run_depend>
```

Comment:
On Wed, Aug 24, 2016 at 02:54:27PM -0700, Robert Haschke wrote:
> > +  <run_depend version_gt="0.3.1">srdfdom</run_depend>
> 
> But only with the to-be-released srdfdom 0.3.2 the SRDFWriter will be in srdfdom. 0.3.1 will not work anymore.

`version_gt` = `>`
`version_gte` = `>='

The patch line above actually requires 0.3.2 or higher.